.github/actions/deploy-with-medic-conf/action.yml

```diff
@@ -0,0 +1,33 @@
+name: 'Deploy with medic-conf'
+inputs:
+  directory:
+    description: The path to the directory containing the configuration to be deployed.
+    required: true
+    default: '.'
+  
+  hostname:
+    description: The hostname of the instance to receive the configuration.
+    required: true
+
+  username:
+    description: The username of an administrator account on the instance at "inputs.hostname". This user will make the deployment.
+    required: true
+    default: 'medic'
```

Comment:
i removed the default value for username since it was medic-specific
but i think the default for directory makes sense. "required" can mean "not empty" and a smart default still seems useful/helpful.

Reply:
Semantically it's an oxymoron, but it's obvious what will happen, so it's not a problem!